packages/flutter_tools/lib/src/commands/build_web.dart

```diff
@@ -67,6 +67,11 @@ class BuildWebCommand extends BuildSubCommand {
       help: 'Sets the optimization level used for Dart compilation to JavaScript. '
           'Valid values range from O0 to O4.'
     );
+    argParser.addOption(
+        'output-dir',
```

Comment:
can you:

1. make this a helper method on the FlutterCommand class `usesOutputDir()` and share it with build_aar
2. make the option name `output` (to support the legacy build_aar use case, you can set `aliases: 'output-dir',`) and `abbr: 'o'` which will allow either --output=/path/to/dir or -o /path/to/dir

Reply:
Good point, I was looking for somewhere upstream that I can add this to work for both `aar` and `web` builds. This should allow us to incrementally add for all the build subcommands